nist/int.go

```diff
@@ -346,6 +343,18 @@ func (i *Int) BigEndian(min, max int) []byte {
 	return buf
 }
 
+// Set value to a number represented by a byte string.
+// Endianness depends on the endianess set in i.
+func (i *Int) SetBytes(a []byte) abstract.Secret {
+	var buff = a
+	if i.Bo == LittleEndian {
+		buff = make([]byte, len(a))
+		util.Reverse(buff, a)
```

Comment:
> Note that the trivial way to fix util.Reverse() to have this behavior is for it simply to call util.Grow() on its first argument.

Well not quite because `util.Grow` still expect a non-nil slice ...